src/session/session_api.c

```diff
@@ -184,7 +184,8 @@ __wt_session_release_resources(WT_SESSION_IMPL *session)
 /*
  * __session_clear_commit_queue --
  *	We're about to clear the session and overwrite the txn structure.
- *	Remove ourselves from the commit timestamp queue if we're on it.
+ *	Remove ourselves from the commit timestamp queue and the read
+ *	timestamp queue if we're on either of them.
  */
 static void
 __session_clear_commit_queue(WT_SESSION_IMPL *session)
```

Comment:
I moved this code to `txn_timestamp.c` and renamed the function `__wt_txn_clear_timestamp_queues` to match the new functionality.